### PR TITLE
FT-335 pick up the product integrated support bundle(drop auth.log) when the pf9ctl CLI operations fail.

### PR DESCRIPTION
**Pull request by Rohitrajak1807**
- dropping `auth.log` from support bundle.
- collecting files in `/var/opt/pf9` as in `express-cli`.
TeamCity build link: https://teamcity.platform9.horse/viewLog.html?buildId=1823945&buildTypeId=Pf9project_Platform9Components_Pf9ctl&tab=buildLog&_focus=1720#_state=125,1712,1720

Diffs for the current release(left) vs the changed(right)
Screenshot 2022-01-31 at 10 25 23_long
Diffs for changed(left) vs express-cli(right)
Screenshot 2022-01-31 at 11 21 47_long

### CODE REVIEW

File: pkg/supportBundle/supportBundle.go [hunk 1]
```diff
@@ -237,8 +242,8 @@ func GenSupportBundle(exec cmdexec.Executor, timestamp time.Time, isRemote bool)
 		// Generate supportBundle if any of Etc / var logs are present or both
 		if errEtc == nil || errVar == nil {
```
Comment (devidask27) [inline pkg/supportBundle/supportBundle.go:243]: Needs to add errOpt check as well.

**Review by devidask27 — CHANGES_REQUESTED** [review]

File: pkg/supportBundle/supportBundle.go (hunk 1 above)
Comment (Rohitrajak1807) [reply pkg/supportBundle/supportBundle.go:243]: Ok, on it

Comment (Rohitrajak1807) [reply pkg/supportBundle/supportBundle.go]: @devidask27  Its done

**Review by devidask27 — APPROVED** [review]
Looks good

**Comment by Rohitrajak1807** [comment]
> Looks good

So, do I proceed to squash and merge

**Review by anupbarve — APPROVED** [review]

**Review by nikhilbandi — APPROVED** [review]
Changes look good.

**Pull request merged**